packages/core/rollup.config.npm.mjs

```diff
@@ -3,21 +3,29 @@ import commonjs from '@rollup/plugin-commonjs'
 import nodeResolve from '@rollup/plugin-node-resolve'
 import typescript from '@rollup/plugin-typescript'
 
-import createRollupConfig, { sharedOutput } from "../../.rollup/index.mjs"
+import { sharedOutput } from "../../.rollup/index.mjs"
+
+const extensions = ['.js', '.ts']
 
 const plugins = [
   nodeResolve({
-    browser: true
+    extensions
   }),
   commonjs(),
-  typescript(),
-  babel({ babelHelpers: 'bundled' }),
+  babel({
+    babelHelpers: 'bundled',
+    exclude: 'node_modules/**',
+    extensions,
+  }),
+  typescript({
+    noForceEmit: true,
+  }),
 ]
 
 const external = [/node_modules/]
 
 export default [
-  createRollupConfig({
+  {
```

Comment:
`createRollupConfig` is becoming a problem now. Basically we're overriding everything anyway, except you can't override the `typescript` plugin it adds

Reply:
I think it was basically used to create config for anything _except_ the core package in the performance repo. if it's not working don't feel like we have to make it work.